src/core/static/wasm/bergamot-translator-worker.appendix.js

```diff
@@ -17,23 +18,20 @@ addOnPreMain(function() {
 
     const languagePair = `${from}${to}`;
 
-    // Load new model if not the same from/to language pair is already loaded
-    const loadedLanguagePair = `${modelFrom}${modelTo}`;
-    if (loadedLanguagePair !== languagePair) {
-      const start = Date.now();
+    // Delete previous instance if a model is already loaded
+    if (model) {
+      model.delete();
+    }
 
-      // Delete previous instance if another model was loaded
-      if (model) {
-        model.delete();
-      }
+    const loadModelStart = performance.now();
 
-      // Vocab files are re-used in both translation directions
-      const vocabLanguagePair = from === "en" ? `${to}${from}` : languagePair;
+    // Vocab files are re-used in both translation directions
+    const vocabLanguagePair = from === "en" ? `${to}${from}` : languagePair;
```

Comment:
Yes, in the future this will be removed, but the code here corresponds to the filesystem embedded in the bergamot-translator build that is currently in use.